SPECS/blktap.spec

```
@@ -156,6 +158,13 @@ fi
 %{?_cov_results_package}
 
 %changelog
+* Fri Feb 02 2024 Gael Duperrey <gduperrey@vates.tech> - 3.37.4-3.1
+- sync with hotfix XS82ECU1056
```

Ideally with caps as we do usually: 
```suggestion
- Sync with hotfix XS82ECU1056
```